### DISCUSSION TITLE
[Feature] <Shift Select for Pictures Not in any Album>

### DISCUSSION THREAD

**Discussion by adromir** [immich-app/immich#11093] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would love it if was possible to Select multiple Pictures with Shift+Mousebutton when browsing pictures not being in any Album, like you can in the normal picture overview 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#9159
